hsds/domain_sn.py

```diff
@@ -624,6 +624,131 @@ async def getScanTime(app, root_id, bucket=None):
     return root_scan
 
 
+async def POST_Domain(request):
+    """ return object defined by h5path list """
+
+    log.request(request)
+    app = request.app
+    params = request.rel_url.query
+    log.debug(f"POST_Domain query params: {params}")
+
+    include_links = False
+    include_attrs = False
+    follow_soft_links = False
+    follow_external_links = False
+
+    if "include_links" in params and params["include_links"]:
```

Comment:
This needs to check for `parent_id` for relative path searches, and use `parent_id` if it exists for `getObjectIdByPath`

Reply:
fixed